store/helper/helper.go

```diff
@@ -293,22 +295,39 @@ func (h *Helper) FetchRegionTableIndex(metrics map[uint64]RegionMetric, allSchem
 			logutil.BgLogger().Error("locate region failed", zap.Error(err))
 			continue
 		}
+		regionsInfo := []*RegionInfo{
+			{
+				ID:       int64(regionID),
+				StartKey: bytesKeyToEncodeHex(region.StartKey),
+				EndKey:   bytesKeyToEncodeHex(region.EndKey),
+			}}
+
+		regionsTableInfos := h.ParseRegionsTableInfos(regionsInfo, tables)
+
+		if tableInfos, ok := regionsTableInfos[int64(regionID)]; ok {
+			if len(tableInfos) == 0 {
+				hotTables = append(hotTables, t)
+				continue
+			}
+			for _, tableInfo := range tableInfos {
+				t.DbName = tableInfo.DB.Name.O
```

Comment:
Calling the `String()` method is more readable.

```suggestion
				t.DbName = tableInfo.DB.Name.String()
```